spec/person_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new(15, 'Joe')
+  end
+  it 'Should return a person object' do
+    expect(@person).to be_an_instance_of Person
+  end
+
+  describe '#age' do
+    it 'Should return Person age' do
+      expect(@person.age).to eql 15
+    end
+  end
+
+  describe '#name' do
+    it 'Should return Person name' do
+      expect(@person.name).to eql 'Joe'
+    end
+  end
+end
```

Comment:
- [ ] Good job so far, but you need to add some tests and you are good to go 🙃 😁:
  - The context of when creating a new instance of the Person class and the constructor method sets a default value for the optional name parameter if it is not provided 😁 
  - The context of when adding new rentals
    -  The scenario of the add_rental method returns a rental and adds it to the book
    - The scenario of the constructor method returns a Person object with provided age, name, and empty rentals
  - The context when checking permissions:
    - The scenario of the `can_use_services?` method returns false if under age and no parent permission
    - The scenario of the `can_use_services?` method returns true if over age
    - The scenario of the `can_use_services?` method returns true if has parent permission